extension/package.json

```diff
@@ -618,6 +623,10 @@
       {
         "fileMatch": "*.dts.json",
         "url": "./syntaxes/dtsDictionarySyntax.json"
+      },
+      {
+        "fileMatch": "template.json",
```

Comment:
We should change this to `al.template.json` less risk of collision and more distinguished. There's also an option to add a wildcard in case of different templates like `*.al.template.json`. You are the best one to judge if that would have any value.

Reply:
I like `al.template.json` 👍

I cannot imagine a scenario where I want to apply different templates on the same set of files, so I think we go for `al.template.json` until we find a real scenario.

Any more thoughts on the names in the json syntax?

Reply:
It is a bit generic, in a good way. My background probably gives me an advantage here. With docs and sample/walk through  I think it will make sense. Maybe it's time to fire up the Wiki for this repo :thinking:

Reply:
The section for this in the readme file will be a bit scarce, I think we'll put it on the wiki later instead. 👍